components/ImgPreviewer.vue

```diff
@@ -10,12 +10,12 @@
     ]"
   >
     <img
-      v-if="computedFileType === 'image'"
+      v-if="computedFileType.includes('image')"
```

Comment:
Change `computedFileType ` to consistently be `mimeType`, then check with `startsWith('image/')`?